### PR TITLE
Rubyday 2019

### PR DESCRIPTION
**Pull request by mukkoo**
rubyday 2019 conference

### CODE REVIEW

File: _data/upcoming.yml [hunk 1]
```diff
@@ -62,6 +62,13 @@
   cfp_phrase: "CFP closes"
   cfp_date: "February 24, 2019"
 
+- name: RubyDay
+  location: Verona, Italy
+  dates: "April 10-11, 2019"
+  url: https://2019.rubyday.it/
+  twitter: rubydayit
+  cfp_date: "February 15, 2019"
```
Comment (jonallured) [inline _data/upcoming.yml]:
```suggestion
  cfp_phrase: "CFP closes"
  cfp_date: "February 15, 2019"
```

**Comment by jonallured** [comment]
Hi @mukkoo thanks for the PR - I just made one minor suggestion so that the site reads nicely. ❤️

**Comment by mukkoo** [comment]
@jonallured done! thank you

**Comment by jonallured** [comment]
Thanks for the PR - I just deployed the site!! 😄

**Comment by mukkoo** [comment]
🎉

**Pull request merged**